iocore/dns/DNS.cc

```diff
@@ -1553,32 +1553,56 @@ dns_process(DNSHandler *handler, HostEnt *buf, int len)
     goto Lerror;
   }
 
+  // Logs using SiteThrottled* version is helpful for noisy logs, being used here
+  // instead of print statements to help with the possible retries when a dns code occurs
   if (h->rcode != NOERROR || !h->ancount) {
     Debug("dns", "received rcode = %d", h->rcode);
     switch (h->rcode) {
     default:
-      Warning("Unknown DNS error %d for [%s]", h->rcode, e->qname);
+      SiteThrottledNote("Unknown: DNS error %d for [%s]", h->rcode, e->qname);
```

Comment:
Maybe this should be kept a `SiteThrottledWarn` actually since this is unexpected.

Reply:
I've changed the default to SiteThrolledWarning.